numpy/polynomial/_polybase.py

```diff
@@ -123,6 +123,10 @@ def _line(self):
         pass
 
     @abstractmethod
+    def _interp(self):
+        pass
```

Comment:
What do you recommend as an alternative to an API change? Why does changing the API affect people subclassing `PolynomialBase`. Sorry, I am not yet an expert developer in Python, just self-taught, and don't quite have the experience necessary to figure out why it might affect someone to add a method to this class.